src/travel_tracker_service.js

```diff
@@ -69,9 +65,43 @@ async function getTravelerDestinations(travelerTrips, traveler) {
   return { travelerTrips, traveler, travelerDestinations };
 }
 
+
+async function getAllTripsForAgent() {
+  const url =
+    "https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/trips/trips";
+  let response = await fetch(url);
+  let awaitedResponse = await response.json();
+  let trips = awaitedResponse.trips;
+  let allTrips = [];
```

Comment:
can you not just pull the trips array out of the response?

Looks like you are just doing a for loop, taking trips out of an array, and putting them back in?

Reply:
Thanks for noticing this ^. Corrected it.